building server in universal story - Error: Can't resolve 'ts-loader'

### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
@angular/cli: 1.4.5
node: 8.6.0
os: linux x64
@angular/animations: 4.4.4
@angular/common: 4.4.4
@angular/compiler: 4.4.4
@angular/core: 4.4.4
@angular/forms: 4.4.4
@angular/http: 4.4.4
@angular/platform-browser: 4.4.4
@angular/platform-browser-dynamic: 4.4.4
@angular/platform-server: 4.4.4
@angular/router: 4.4.4
@angular/cli: 1.4.5
@angular/compiler-cli: 4.4.4
@angular/language-service: 4.4.4
typescript: 2.3.4

### Repro steps.
1. Follow the universal story to setup existing angular-cli project to universal.
2. Run the command to build the universal server. 
`webpack --config webpack.server.config.js --progress --colors`

### The log given by the failure.
> webpack --config webpack.server.config.js --progress --colors

Hash: eb41d9b8aea4d4f2ca58              
Version: webpack 3.6.0
Time: 32ms

ERROR in Entry module not found: Error: Can't resolve 'ts-loader' in <some project>
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! poc@0.0.0 webpack:server: `webpack --config webpack.server.config.js --progress --colors`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the poc@0.0.0 webpack:server script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/eyal/.npm/_logs/2017-10-11T05_21_36_973Z-debug.log
eyal@eyal-ThinkPad-X270:~/poc$ npm i ts-loader
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ ts-loader@2.3.7
added 115 packages in 6.207s

--- a/docs/documentation/stories/universal-rendering.md
+++ b/docs/documentation/stories/universal-rendering.md
@@ -20,10 +20,10 @@ This story will show you how to set up Universal bundling for an existing `@angu
 
 Install `@angular/platform-server` into your project. Make sure you use the same version as the other `@angular` packages in your project.
 
-> You'll also need @nguniversal/module-map-ngfactory-loader, as it's used to handle lazy-loading in the context of a server-render. (by loading the chunks right away)
+> You'll also need ts-loader (for your webpack build we'll show later) and @nguniversal/module-map-ngfactory-loader, as it's used to handle lazy-loading in the context of a server-render. (by loading the chunks right away)
 
 ```bash
-$ npm install --save @angular/platform-server @nguniversal/module-map-ngfactory-loader
+$ npm install --save @angular/platform-server @nguniversal/module-map-ngfactory-loader ts-loader
 ```
 
 ## Step 1: Prepare your App for Universal rendering

